ui/partials/perf/graphsctrl.html

```diff
@@ -24,7 +27,7 @@
         </tr>
       </table>
       <button class="btn btn-primary" ng-click="addTestData()">
-        <span class="glyphicon glyphicon-plus" aria-hidden="true"></span> Add more test data</button>
+        <span class="glyphicon glyphicon-plus" aria-hidden="true"></span> Add <span ng-show="seriesList.length > 0">more</span> test data</button>
     </div>
     <div id="data-display">
       <select ng-model="myTimerange" ng-options="timerange.text for timerange in timeranges track by timerange.value" ng-change="timeRangeChanged()">
```

Comment:
While we're here, let's disable this element if the seriesList is empty. We can use ngDisabled for this:

https://docs.angularjs.org/api/ng/directive/ngDisabled

basically I think we just need to add this to the select element:

ng-disabled="seriesList.length == 0"